src/main/resources/config.yml

```
@@ -31,6 +31,8 @@ researches:
   free-in-creative-mode: true
   enable-fireworks: true
   disable-learning-animation: false
+  enable-vault-economy: false
+  economy-price-multiplier: 1000.0
```

I think this should be grouped and simplified
```suggestion
  vault-integration
    enabled: false
    cost-multiplier: 1000.0
```

---

do you mean you want `vault-integration` to be inside `researches`? or you want it at the top level?